src/settings-ui/Settings.UI/SettingsXAML/Views/EnvironmentVariablesPage.xaml

```diff
@@ -27,17 +27,13 @@
                 <controls:SettingsGroup x:Uid="EnvironmentVariables_Activation_GroupSettings" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
                     <controls1:SettingsCard
                         x:Uid="EnvironmentVariables_LaunchButtonControl"
-                        ActionIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily},
-                                                 FontSize=14,
-                                                 Glyph=&#xE8A7;}"
+                        ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
```

Comment:
@niels9001 for some reason, this one - and only this one - reverts back to MDL2. Do you know why?